tools/docker/syzbot/Dockerfile

```diff
@@ -66,6 +66,14 @@ RUN test "$(uname -m)" != x86_64 && exit 0 || \
     qemu-utils qemu-system-misc qemu-system-x86 qemu-system-arm qemu-system-aarch64  \
     qemu-system-s390x qemu-system-mips qemu-system-ppc
 
+# Install gcloud https://cloud.google.com/sdk/docs/install#deb.
+RUN apt-get install -y -q apt-transport-https ca-certificates gnupg curl
```

Comment:
We already install everything except for ca-certificates before this.

Reply:
Duplicates cost us nothing here.
I don't want the dependencies removal at the beginning of this file to affect this part.

Reply:
But we now install curl 3 times + install only missing deps here:
RUN apt-get install -y -q gnupg software-properties-common apt-transport-https
This looks like a mess. Would be better to install it once but properly at the top, and then use it. Maybe on a separate line with a comment.

Reply:
If you remove that line, we cam move forward with this change.
Curl is already used in this script before, so using it more in the same way it's already used is not objectionable.

Reply:
Thanks. Yes, I'll switch back to this track today and do it.
I have parallel track with authorization changes.

Reply:
Done. PTAL.